docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -13,99 +13,208 @@ description: guide for doing a lift and shift migration of airflow to dagster
   !
 </Note>
 
-Dagster can convert your Airflow DAGs into Dagster jobs, enabling a lift-and-shift migration from Airflow to Dagster without any rewriting.
+Dagster can convert your Airflow DAGs into Dagster jobs, enabling a lift-and-shift migration from Airflow without any rewriting.
 
-This guide will walk you through the steps of doing this migration.
+This guide will walk you through the steps of performing this migration.
 
 ---
 
 ## Prerequisites
 
-To get started, you will need to install the `dagster` and `dagster-airflow` Python packages:
+Migrations are complicated as there are lots of moving pieces. It's important for you to go in knowing exactly how much scope and complexity you'll be tackling.
```

Comment:
I don't think we need to scare people right out of gate.

Suggestion:

"Before beginning a migration it is important for you to do some upfront analysis:"

Then drop, "To complete the migration, you'll need:"